### PR TITLE
Observability plugin support

### PR DESCRIPTION
**Pull request by umarali-nagoor**
Test Results:

Screenshot 2021-03-24 at 10 14 13 PM
Screenshot 2021-03-24 at 9 50 08 PM
Screenshot 2021-03-24 at 9 08 12 PM
Screenshot 2021-03-24 at 7 09 49 PM

### CODE REVIEW

File: ibm/resource_ibm_ob_logging.go [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+	clusterName := parts[0]
+	loggingID := parts[1]
+
+	targetEnv, err := getLoggingTargetHeader(d, meta)
+	if err != nil {
+		return err
+	}
+
+	config, err := client.Logging().GetLoggingConfig(clusterName, loggingID, targetEnv)
+	if err != nil {
```
Comment (hkantare) [inline ibm/resource_ibm_ob_logging.go:221]: Inside the err check if resp.statuscode is 404 then set d.SetID("") and return nil
else retrun err

File: ibm/resource_ibm_ob_logging.go [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
+	loggingID := parts[1]
+
+	//populate logging logDNA configure create request
+	params := v2.LoggingDeleteRequest{
+		Cluster:  clusterName,
+		Instance: loggingID,
+	}
+
+	_, err = client.Logging().DeleteLoggingConfig(params, targetEnv)
+	if err != nil {
```
Comment (hkantare) [inline ibm/resource_ibm_ob_logging.go:325]: same here 
Inside the err check if resp.statuscode is 404 then set d.SetID("") and return nil
else retrun err

File: ibm/resource_ibm_ob_logging.go [hunk 3]
```diff
@@ -0,0 +1,358 @@
...
+		return err
+	}
+	d.SetId("")
+	return nil
+
+}
+
+func resourceIBMLoggingExists(d *schema.ResourceData, meta interface{}) (bool, error) {
```
Comment (hkantare) [inline ibm/resource_ibm_ob_logging.go:333]: This exists method is no longer required

File: ibm/resource_ibm_ob_monitoring.go [hunk 4]
```diff
@@ -0,0 +1,366 @@
...
+	clusterName := parts[0]
+	monitoringID := parts[1]
+
+	targetEnv, err := getMonitoringTargetHeader(d, meta)
+	if err != nil {
+		return err
+	}
+
+	config, err := client.Monitoring().GetMonitoringConfig(clusterName, monitoringID, targetEnv)
+	if err != nil {
```
Comment (hkantare) [inline ibm/resource_ibm_ob_monitoring.go:227]: same comments as above

**Comment by hkantare** [comment]
Add the resoectie docs for the new resources

**Comment by umarali-nagoor** [comment]
Test Results after review comments addressed

Screenshot 2021-03-26 at 7 33 24 PM
Screenshot 2021-03-26 at 7 33 17 PM
Screenshot 2021-03-26 at 7 31 24 PM
Screenshot 2021-03-26 at 7 30 59 PM

**Pull request merged**